src/routes/webhooks/paddle.ts

```diff
@@ -97,6 +163,23 @@ export const paddle = async (fastify: FastifyInstance): Promise<void> => {
                   data: eventData,
                   state: false,
                 });
+                await logPaddleAnalyticsEvent(
+                  eventData,
+                  AnalyticsEventName.CancelSubscription,
+                );
+                break;
+              case EventName.SubscriptionUpdated:
+                await logPaddleAnalyticsEvent(
+                  eventData,
+                  AnalyticsEventName.ChangeBillingCycle,
+                );
+                break;
+              case EventName.TransactionPaid:
+                await logPaddleAnalyticsEvent(
+                  eventData,
+                  AnalyticsEventName.ReceivePayment,
+                );
+                break;
```

Comment:
Maybe we should use TransactionCompleted?
It is the final event that happens
https://developer.paddle.com/webhooks/transactions/transaction-completed